sonic-slave-bookworm/Dockerfile.j2

```diff
@@ -449,7 +449,8 @@ RUN apt-get update && apt-get install -y eatmydata && eatmydata apt-get install
         meson \
         python3-pyelftools \
         hwdata \
-        libzip-dev
+        libzip-dev \
+	libnl-genl-3-dev
```

Comment:
Why need to install it? We build libnl from sources (including libnl-genl-3-dev) - https://github.com/sonic-net/sonic-buildimage/blob/master/rules/libnl3.mk#L20

Reply:
It is required for cmake build. If I run it without it, it fail

Reply:
@dprital The question is why it fails, if libnl-genl-3-dev is built by SONiC there's no need to install it from debian repo. I think there's a dependency on libnl missing in our sdk makefiles